docs/hugo/content/guide/crd-management.md

```diff
@@ -0,0 +1,121 @@
+---
+title: CRD management in ASO
+weight: -3
+---
+
+## How does CRD management work in Azure Service Operator?
+
+ASO performs CRD management differently than many traditional operators. You might notice if you install the
+[Helm chart](../installing-from-helm) or the [YAML](../installing-from-yaml) that there are NO CRDs installed by default.
+
+This is for two reasons:
+
+1. ASO supports so many CRDs that it exceeds the maximum Helm chart size when CRDs are included, so the CRDs cannot be 
+   included in the Helm chart.
+2. ASO supports so many CRDs that it can cause problems if they're all installed onto a Kubernetes cluster with a low
+   memory limit. See [#2920](https://github.com/Azure/azure-service-operator/issues/2920) for details.
+
+For the above reasons, by default each version of the ASO operator pod embeds the CRDs inside the pod and installs them when
+the pod launches. 
+
+**No CRDs are installed by default.** New CRDs will only be installed if they are specified in the 
+`--crd-pattern` cmdline argument to the `manager` container. `--crd-pattern` is a semicolon delimited string of CRD
+patterns to apply. 
+
+**Existing CRDs are always upgraded.** If a CRD already exists in the cluster (from a previous version of ASO or 
+a manual CRD installation), it will always be updated to the latest version of the CRD.
+
+## With Helm
+
+### Automatic CRD installation (recommended)
+
+Use the `crdPattern` value in `values.yaml` to specify what new CRDs to install. Existing CRDs will always be upgraded
+even if `crdPattern` is empty.
+
+Example: `resources.azure.com/*;containerservice.azure.com/*;keyvault.azure.com/*;managedidentity.azure.com/*;eventhub.azure.com/*`
```

Comment:
Now I see this, I'm wondering about verbosity of the repetition of `.azure.com` in every pattern - what if we tweaked things 
```suggestion
Example: `resources/*;containerservice/*;keyvault/*;managedidentity/*;eventhub/*`
```

If we do it now (before the v2.1.0 release) we can make this the canonical form. If we do it post release, it would be an alternative only.

Reply:
Hmm... yeah I agree this would be nice. It's easy to do. I'll make the change

Reply:
As discussed on teams, actually leaving this as-is for now and we can revisit if folks complain. What we have is unambiguous and matches the first line of all of our CRD samples. Saving users this typing here would be nice but since we can't save them from having to type it elsewhere erring on the side of consistency for now until we get other data.